docs/development/manufacturer/manufacturer-design-guidelines.md

```diff
@@ -460,20 +461,22 @@ Importantly, although the Bidirectional DShot ENABLED may require lower PID Loop
 
 Looptime and Performance Recommendation Table:
 
-| MCU                           | IMU                                   | Sampling Rate | Bidirectional DShot Status | PID Loop Rate | DShot Protocol |
-| :---------------------------- | :------------------------------------ | :------------ | :------------------------- | :------------ | :------------- |
-| F7X2, H7XX, G4XX, and similar | MPU60X0, ICM2060X, ICM42688P          | 8 kHz         | Enabled or Disabled        | 8 kHz         | DShot 600      |
-|                               | BMI-270                               | 3.2 kHz       | Enabled or Disabled        | 3.2 kHz       | DShot 300      |
-| F405                          | MPU6000, MPU6050, ICM20601, ICM42688P | 8 kHz         | Enabled                    | 4 kHz         | DShot 300      |
-|                               | MPU6000, MPU6050, ICM20601, ICM42688P | 8 kHz         | Disabled (not recommended) | 8 kHz         | DShot 600      |
-|                               | BMI-270                               | 3.2 kHz       | Enabled or Disabled        | 3.2 kHz       | DShot 300      |
-| F411 UART Rx \*\*             | MPU6000, MPU6050, ICM20601            | 8 kHz         | Enabled                    | 4 kHz         | DShot 300      |
-|                               | MPU6000, MPU6050, ICM20601            | 8 kHz         | Disabled (not recommended) | 8 kHz         | DShot 600      |
-|                               | BMI-270                               | 3.2 kHz       | Enabled                    | 3.2 kHz       | DShot 300      |
-| F411 SPI Rx \*\*\*            | MPU6000, MPU6050, ICM20601            | 8 kHz         | Enabled                    | 2 kHz         | DShot 300      |
-|                               | MPU6000, MPU6050, ICM20601            | 8 kHz         | Disabled (not recommended) | 4 kHz         | DShot 300      |
-|                               | BMI-270                               | 3.2 kHz       | Enabled                    | 1.6 kHz       | DShot 300      |
-|                               | BMI-270                               | 3.2 kHz       | Disabled (not recommended) | 3.2 kHz       | DShot 300      |
+| MCU                           | IMU                          | Sampling Rate | Bidirectional DShot Status | PID Loop Rate | DShot Protocol |
+| :---------------------------- | :--------------------------- | :------------ | :------------------------- | :------------ | :------------- |
+| H7XX                          | MPU60X0, ICM2060X, ICM42688P | 8 kHz         | Enabled or Disabled        | 8 kHz         | DShot 600      |
```

Comment:
while i understand this cleans up the tables, does it make searching harder?

Reply:
Other rows already used - so made it consistent.
But we need to decide on other factors like @ot0tot mentioned.